src/renderer/components/TagsTabBar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Tag, Text } from "@ledgerhq/react-ui";
+import { Flex } from "@ledgerhq/react-ui/components/layout";
+
+type Props = {
+  onSelectedChanged: (index: number) => any;
+  selectedIndex: number;
+  tabs: string[];
+};
+
+const TagsTabBar = (props: Props) => {
+  const { onSelectedChanged, selectedIndex, tabs } = props;
+  return (
+    <Flex flexDirection="row">
+      {tabs.map((tab, index) => {
+        const isSelected = index === selectedIndex;
+        const color = isSelected ? "palette.neutral.c100" : "palette.neutral.c80";
+        return (
+          <Tag
+            type="opacity"
+            key={index}
+            onClick={() => onSelectedChanged(index)}
+            active={isSelected}
+            style={{ marginLeft: index === 0 ? 0 : "8px" }}
```

Comment:
could you create a PR to fix the tag cursor :thinking: 
also we could use the proxified styled on the ui lib to control margin left inline with no style prop needed
(nb for the cursor you can actually target in styled components props things like "onClick" )

Reply:
Actually someone made similar a tab bar in UI repo since I wrote this code weeks ago so I'll use it instead